app/controllers/collections_controller.rb

```diff
@@ -158,20 +158,36 @@ def ensure_correct_collection_provider
   end
 
   def prepare_translated_collections
-    original_collection_native_xml = cmr_client.get_concept(params[:id],token, {})
-    return { error: 'Failed to retrieve collection from CMR' } unless original_collection_native_xml.success?
+    cmr_response_original = cmr_client.get_concept(params[:id],token, {})
+    return { error: 'Failed to retrieve collection from CMR' } unless cmr_response_original.success?
 
-    content_type = original_collection_native_xml.headers.fetch('content-type').split(';')[0]
+    content_type = cmr_response_original.headers.fetch('content-type').split(';')[0]
     return { error: 'This collection is already in UMM format so there is no loss report' } if content_type.include?('application/vnd.nasa.cmr.umm+json')
 
-    translated_collection_native_xml = cmr_client.translate_collection(JSON.pretty_generate(@collection), "application/#{Rails.configuration.umm_c_version}; charset=utf-8", content_type,  skip_validation=true)
-    return { error: 'Failed to translate collection from UMM back to native format' } unless translated_collection_native_xml.success?
+    cmr_response_translated = cmr_client.translate_collection(JSON.pretty_generate(@collection), "application/#{Rails.configuration.umm_c_version}; charset=utf-8", content_type,  skip_validation=true)
+    return { error: 'Failed to translate collection from UMM back to native format' } unless cmr_response_translated.success?
```

Comment:
I think we should log the collections which cause these errors.  It will make it easier to debug in the future.  See the logging for a failed publish attempt (collection draft controller) for guidance.